ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml

```diff
@@ -97,12 +97,7 @@ tests:
     test:
     - as: scorecard-test
       cli: latest
-      commands: |
-        set -x
-        curl -Lso /tmp/operator-sdk.tar.gz https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/operator-sdk/4.15.21/operator-sdk-v1.31.0-ocp-linux-x86_64.tar.gz
-        ls -al /tmp/operator-sdk.tar.gz
-        tar -xvzf /tmp/operator-sdk.tar.gz --strip=2 -C /tmp
-        /tmp/operator-sdk scorecard bundle -o text --kubeconfig $KUBECONFIG -n default
+      commands: make scorecard-test KUBECONFIG=$KUBECONFIG
```

Comment:
I think you can probably drop `KUBECONFIG=$KUBECONFIG`.  From [GNU Make docs][1]:

> Every environment variable that make sees when it starts up is transformed into a make variable with the same name and value.

And [from POSIX][2]:

> Macro definitions shall be taken from the following sources, in the following logical order, before the makefile(s) are read.
> ...
> 3. The contents of the environment, excluding the MAKEFLAGS and SHELL variables and including the variables with null values.
> ...

[1]: https://www.gnu.org/software/make/manual/html_node/Environment.html
[2]: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/make.html#tag_20_76_13_05